src/raw/mod.rs

```diff
@@ -1312,6 +1224,19 @@ impl<A: Allocator + Clone> RawTableInner<A> {
         Bucket::from_base_index(self.data_end(), index)
     }
 
+    #[cfg_attr(feature = "inline-more", inline)]
+    unsafe fn bucket_ptr(&self, index: usize, size_of: usize) -> *mut u8 {
```

Comment:
This function absolutely needs to be `#[inline]` at minimum, it is not generic.

Reply:
It is generic over `A`, so I figured this should match the inlining of `fn bucket`. Though I would put `#[inline(always)]` on both of them (on release mode).

Reply:
There's a bunch of small functions which are beneficial to inline. I'll open a PR for those. It seems the compile time benefit of https://github.com/rust-lang/hashbrown/pull/119 mostly comes from removing inline on `get`, `insert`, etc.